app/controllers/idv/in_person/usps_locations_controller.rb

```diff
@@ -65,6 +65,7 @@ def handle_error(err)
           Faraday::TimeoutError => :unprocessable_entity,
           Faraday::BadRequestError => :unprocessable_entity,
           Faraday::ForbiddenError => :unprocessable_entity,
+          ActionController::InvalidAuthenticityToken => :unprocessable_entity,
```

Comment:
could we add a controller-level

```ruby
protect_from_forgery with: :reset_session
```

so that it never throws an error in the first place?

https://api.rubyonrails.org/classes/ActionController/RequestForgeryProtection/ClassMethods.html#method-i-protect_from_forgery

Reply:
I'm not sure on the best strategy since this is an API and mostly copied what's already here, I'm primarily wanting to not return a 500. I'd defer to the folks on Joy.